cluster/addons/e2e-rbac-bindings/random-addon-grabbag.yaml

```diff
@@ -1,20 +0,0 @@
-# TODO remove this
-# currently, the kube-addon-manager is adding lots of pods which all share
-# the system:serviceaccount:kube-system:default identity.  We need to subdivide
-# those service accounts, figure out which ones we're going to make bootstrap roles for
-# and bind those particular roles in the addon yaml itself.  This just gets us started
-apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
does removing this mean the add-on manager will delete the existing binding on an upgrade? we should not do that automatically. what do we need to do to avoid binding to the default service account in a new cluster, but avoid deleting the existing binding on an upgrade?